clients/da-vinci-client/build.gradle

```diff
@@ -17,13 +17,24 @@ dependencies {
     exclude group: 'xerces', module: 'xerces' // resolve conflicting transitive dependency
     exclude group: 'com.fasterxml.jackson.core', module: 'jackson-core'// resolve conflicting transitive dependency
   }
-
+  implementation('org.apache.helix:helix-core:1.4.1:jdk8') {
```

Comment:
Please help me understand this statement.
we want `org.apache.helix:helix-core` but we also do not want any of `org.apache.helix` ?

Reply:
Right, previously we used only helix-core as well and only the used library are ported to be JDK8 compatible.